www/src/js/models/snap.js

```diff
@@ -166,10 +166,6 @@ module.exports = Backbone.Model.extend({
       response.icon = this.defaults.icon;
     }
 
-    if (response.hasOwnProperty('developer') && !response.developer.length) {
-      response.developer = this.defaults.developer;
-    }
-
```

Comment:
I don't know much about flow of the application, but what would happen now if developer is not set and we don't set it to default?